Code/Framework/AzCore/Tests/Debug/PerformanceCollectorTests.cpp

```diff
@@ -0,0 +1,215 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzCore/UnitTest/UnitTest.h>
+#include <AzTest/AzTest.h>
+
+#include <AzCore/Debug/PerformanceCollector.h>
+#include <AzCore/JSON/document.h>
+
+using namespace AZ;
+using namespace Debug;
+
+namespace UnitTest
+{
+    class PerformanceCollectorTest
+        : public AllocatorsFixture
+    {
+    public:
+        PerformanceCollectorTest()
+        {
+        }
+
+        void SetUp() override
+        {
+            AllocatorsFixture::SetUp();
+        }
+
+        ~PerformanceCollectorTest() override
+        {
+        }
+
+        void TearDown() override
+        {
+            AllocatorsFixture::TearDown();
+        }
+    }; //class PerformanceCollectorTest
+
+    TEST_F(PerformanceCollectorTest, CreatePerformanceCollector_CollectPerformance_ValidateStatisticalOutput)
+    {
+        // In this test we use the PerformanceCollector to collect statistical data for three parameters.
+        // Statistical data summarizes the results across a single batch of sampled frames.
+        // In this test we run 3 batches, which means we expect 3 rows (aka json object) per measured
+        // parameter. Each json object will contain statistical summary of a batch made of 10 frames.
+
+        const AZStd::string LogCategory("PerformanceCollectorTest");
+        constexpr AZStd::string_view PerfParam1("param1");
+        constexpr AZStd::string_view PerfParam2("param2");
+        constexpr AZStd::string_view PerfParam3("param3");
```

Comment:
Why is the LogCategory not a string_view as well?

Reply:
```suggestion
        constexpr AZStd::string_view LogCategory("PerformanceCollectorTest");
        constexpr AZStd::string_view PerfParam1("param1");
        constexpr AZStd::string_view PerfParam2("param2");
        constexpr AZStd::string_view PerfParam3("param3");
```

Reply:
Fixed.